assignments/objects.js

```diff
@@ -18,28 +18,69 @@ const example = {
 }
 
 // Write your intern objects here:
+const obj1 = {
+  "id": 0,
+  "name": "Mitzi",
+  "email": "mmelloy0@psu.edu",
+  "gender": "F"
+}
+
+const obj2 = {
+  "id": 1,
+  "name": "Kennan",
+  "email": "kdiben1@tinypic.com",
+  "gender": "M"
+}
 
+const obj3 = {
+  "id": 2,
+  "name": "Keven",
+  "email": "kmummery2@wikimedia.org",
+  "gender": "M"
+}
+
+const obj4 = {
+  "id": 3,
+  "name": "Gannie",
+  "email": "gmartinson3@illinois.edu",
+  "gender": "M"
+}
 
+const obj5 = {
+  "id": 4,
+  "name": "Antonietta",
+  "email": "adaine5@samsung.com",
+  "gender": "F"
+}
 // ==== Challenge 2: Reading Object Data ==== 
 // Once your objects are created, log out the following requests from HR into the console:
 
 // Mitzi's name
-
+console.log(obj1["name"])
 // Kennan's ID
-
+console.log(obj2["id"])
 // Keven's email
-
+console.log(obj3["email"])
 // Gannie's name
-
+console.log(obj4["name"])
 // Antonietta's Gender
-
+console.log(obj5["gender"])
 // ==== Challenge 3: Object Methods ==== 
 // Give Kennan the ability to say "Hello, my name is Kennan!" Use the console.log provided as a hint.
 // console.log(kennan.speak());
+function speak () {
+  console.log("Hello, my name is" + this["name"])
+}
+obj2.speak ()
 
 // Antonietta loves math, give her the ability to multiply two numbers together and return the product. Use the console.log provided as a hint.
 //console.log(antonietta.multiplyNums(3,4));
 
+obj5.multiplyNums = function (num1, num2) {
+return num1 * num2
+}
+
```

Comment:
I should have scrolled further down. Exactly like this, but here we used an anonymous function to skip the step of explicitly declaring a named function.